packages/truffle-contract/scripts/test.sh

```diff
@@ -3,7 +3,7 @@
 set -o errexit
 
 if [ "$GETH" == true ]; then
-  mocha --timeout 50000 --grep @geth --colors
+  mocha --timeout 50000 --grep @geth --colors --exit
 else
-  mocha --no-warnings --timeout 7000 --colors
+  mocha --no-warnings --timeout 7000 --colors --exit
```

Comment:
Might be worth a comment so we remember why `--exit` is included in the future

Reply:
This only affects us running in CI against Geth because we can't really shut it down from the tests. Since mocha v4 (I thought it was 5) mocha [doesn't automatically exit](https://boneskull.com/mocha-v4-nears-release/#otherbreakingchanges) if there are open connections even if it *thinks* it's done. Everything works ok with ganache because we close it in an `after`.